rspec-core.gemspec

```diff
@@ -45,14 +44,19 @@ Gem::Specification.new do |s|
     s.add_runtime_dependency "rspec-support", "~> #{RSpec::Core::Version::STRING.split('.')[0..1].concat(['3']).join('.')}"
   end
 
+  s.add_development_dependency "aruba",    "~> 0.14.9"
   s.add_development_dependency "cucumber", "~> 1.3"
   s.add_development_dependency "minitest", "~> 5.3"
-  s.add_development_dependency "aruba",    "~> 0.14.9"
+  s.add_development_dependency "test-unit", "~> 3.0"
+
+  s.add_development_dependency "simplecov", "~> 0.8"
```

Comment:
Wait why are you adding simplecov?

Reply:
To my best knowledge, `add_development_dependency` only affects local bundling. I've moved it over from the Gemfile.
As far as I understand we can shuffle those dependencies between `add_development_dependency` and Gemfile without affecting anything.
To me, it's better to keep those gems that we directly depend on here, while reserving the Gemfile to add version constraints for third-party dependencies like `ffi` and those that require a conditional, e.g.
```
if RUBY_VERSION >= '2.4'
```